galaxykit/command.py

```diff
@@ -302,13 +302,23 @@ def main():
         elif args.kind == "collection":
             if args.operation == "upload":
                 if len(args.rest) == 0:
-                    (namespace, collection_name) = (client.username, None)
-                else:
+                    (namespace, collection_name, version) = (
+                        client.username,
+                        None,
+                        None,
+                    )
+                if len(args.rest) == 2:
                     (namespace, collection_name) = args.rest
+                    version = "1.0.0"
+                else:
```

Comment:
this results in...

0 args:
namespace = None
collection_name = None
version = None

1 arg:
namespace = "arg1"
collection_name = None
version = None

2 args:
namespace = "arg1"
collection_name = "arg2"
version = "1.0.0"

3 args:
namespace = "arg1"
collection_name = "arg2"
version = "arg3"

(the namespace=client.username logic is dead now)

Reply:
Updated to `elif`